vendor.conf

```diff
@@ -33,7 +33,7 @@ github.com/imdario/mergo                            1afb36080aec31e0d1528973ebe6
 golang.org/x/sync                                   cd5d95a43a6e21273425c7ae415d3df9ea832eeb
 
 # buildkit
-github.com/moby/buildkit                            fcb87e6b8ccf3631a65799cc56caa76f9117816e # v0.8.0-rc2
+github.com/moby/buildkit                            950603da215ae03b843f3f66fbe86c4876a6f5a1 # master (v0.9.0-dev)
```

Comment:
Looks like there's no `v0.8` branch yet, so wondering if this would be `v0.8.1-dev` ?

https://github.com/moby/buildkit/compare/v0.8.0...950603da215ae03b843f3f66fbe86c4876a6f5a1

Reply:
generally speaking I believe `-dev` is not derived from patch number versions.

Reply:
that's correct, this is more of a "manual" indication; alternatively `v0.8.0-4-g950603da`; 

```bash
git checkout 950603da215ae03b843f3f66fbe86c4876a6f5a1
git describe --tags --match 'v*'
v0.8.0-4-g950603da
```